.github/workflows/project.yml

```diff
@@ -0,0 +1,20 @@
+name: Add to Project
+
+on:
+  issues:
+    types:
+      - opened
+      - transferred
+  pull_request_target:
+    types:
+      - opened
+
+jobs:
+  add-to-project:
+    name: PR
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/add-to-project@v0.3.0
+        with:
+          project-url: https://github.com/orgs/Arelle/projects/3
+          github-token: ${{ secrets.PROJECT_TOKEN }}
```

Comment:
Before this PR is merged this project should be made public and a token (limited to only project access) needs to be created.

Reply:
Moved this out of this PR so that the issue templates aren't blocked by the workflow/github token.